modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -116,6 +116,7 @@ export interface ConfigurationParameters {
     basePath?: string; // override base path
     fetchApi?: FetchAPI; // override for fetch implementation
     middleware?: Middleware[]; // middleware to apply before/after fetch requests
+    stringify?: (params: HTTPQuery) => string; // stringify function for query strings
```

Comment:
naming should be more specific, as it is only used for query parameter serialization:
```suggestion
    queryParamsStringify?: (params: HTTPQuery) => string; // stringify function for query strings
```

Reply:
thank you for the review, i renamed it.